creme/creme_core/templates/creme_core/detail/entity-filter.html

```diff
@@ -13,28 +14,17 @@
 
 {% block logo %} {% widget_icon name='filter' size='brick-hat-bar' label=_('Filter of Entity') %}{% endblock %}
 
+{# BEWARE : NOT IN A BRICK. Actions that needs a brick reload will not work (redirections/reload are fine) #}
```

Comment:
"(redirections/**reload** are fine)" ?

Reply:
"page reload" would be more accurate :grin: